datadog_checks_base/CHANGELOG.md

```diff
@@ -9,8 +9,11 @@
 
 ***Added***:
```

Comment:
I think this can be deleted and have these changes listed under the the Added header in line 5